app/routes/app/AppRoute.js

```diff
@@ -201,32 +201,27 @@ const mapDispatchToProps = {
 };
 
 export default compose(
-  prepare((_, dispatch) => dispatch(loginAutomaticallyIfPossible()), [], {
-    componentDidMount: false,
-    componentWillReceiveProps: false,
-  }),
-  prepare((_, dispatch) => dispatch(fetchMeta()), [], {
-    componentDidMount: false,
-    componentWillReceiveProps: false,
-    awaitOnSsr: false,
-  }),
-  prepare(
-    (props, dispatch) =>
-      Promise.all([
-        dispatch(fetchNotificationData()),
-        dispatch((dispatch, getState) => {
-          if (!selectIsLoggedIn(getState())) {
-            return Promise.resolve();
-          }
-          return dispatch(
-            fetchMeetings({
-              dateAfter: moment().format('YYYY-MM-DD'),
-            })
-          );
-        }),
-      ]),
-    [],
-    { awaitOnSsr: false }
+  withPreparedDispatch(
+    'login',
+    (_, dispatch) => dispatch(loginAutomaticallyIfPossible()),
+    () => [],
+    { runSync: true }
+  ),
+  withPreparedDispatch('fetchMeta', (_, dispatch) => dispatch(fetchMeta())),
+  withPreparedDispatch('fetchDivData', (props, dispatch) =>
```

Comment:
These can be merged into one as well if the behaviour is as it should without the options on `fetchMeta`

Reply:
Yup